src/instance/InstanceController.ts

```diff
@@ -1432,4 +1432,23 @@ export class ServerController {
 
     res.json(infoData);
   }
+
+  @Get('/commit')
+  @ApiOperation({summary: 'Get information about the server.'})
+  @ApiResponse({
+    schema: {
+      example: 'string',
+    },
+    status: 200,
+  })
+  @ApiResponse(status500description)
+  async getCommit(@Res() res: Response): Promise<void> {
+    try {
+      const commitHashFile = path.join(config.DIST_DIR, 'commit');
```

Comment:
@ffflorian where should I point the `commitHashFile`.. ?

https://github.com/wireapp/wire-web-ets/blob/master/src/routes/commit/commitRoute.ts#L30

From the implementation point of view, it's the same as the old one.. and getting this:

```jsonc
// the result from master
{
  "code": 500,
  "error": "ENOENT: no such file or directory, open '/Users/lipis/work/wire/etc/src/commit'",
  "stack": "Error: ENOENT: no such file or directory, open '/Users/lipis/work/wire/etc/src/commit'"
}
```

Reply:
Yes, the commit file was created when building. You can just catch the error and log it.